index.html

```diff
@@ -1,37 +1,69 @@
-<!-- Markup the content below with tags and attributes such that it can be styled in CSS-->
 <!DOCTYPE html>
 <html lang="en">
-<head>
-<!-- Link your CSS stylesheet -->
-    
-</head>
-<body>
+  <head>    
+  <link rel="stylesheet" href="index.css"
+  </head>
+  <body>
+    <nav>
+      <button>
         Plan your Visit
-        Learn about the Park
+      </button>
+      <button>
+        Learn about the Pack
+      </button>
+      <button>
         Get Involved
-   
-      
-      "header_img.jpg"
-      Yosemite Park Guide
-      Come explore California’s beautiful wilderness.
-      Always open. Reservations are not available or required to enter the park.
-      
-      Not just a great valley, but a shrine to human foresight, the strength of granite, the power of glaciers, the persistence of life, and the tranquility of the High Sierra.
-      
-      First protected in 1864, Yosemite National Park is best known for its waterfalls, but within its nearly 1,200 square miles, you can find deep valleys, grand meadows, ancient giant sequoias, a vast wilderness area, and much more.
-      
-      "https://image.flaticon.com/icons/svg/2979/2979348.svg"
-      Fire is a natural and essential part of Yosemite. We manage fire carefully and study how it interacts with the park’s ecosystems. This blog provides updates about fires in Yosemite. Check for current fire restrictions.
-      
-      "https://image.flaticon.com/icons/svg/820/820374.svg"
-      We strongly recommend that you make reservations for lodging, camping, and backpacking. You can pay the park entrance fee upon arrival (there's no need to pay it in advance).
+      </button3>
+    </nav>
+  </body>
+</html>
+<br>
+<br>
+<br>
+
+
+<section class="marketing-header">
```

Comment:
When marking up with HTML, you'll want to keep all of the elements within the `<body>` element.